pronto/api/signatures/matrices.py

```diff
@@ -13,19 +13,43 @@ def get_matrices(accessions):
     con = utils.connect_pg(utils.get_pg_url())
     cur = con.cursor()
 
-    signatures, comparisons = get_comparisons(cur, accessions)
+    signatures, comparisons, exclusive = get_comparisons(cur, accessions)
 
     cur.close()
     con.close()
     
     return jsonify({
         "signatures": signatures,
-        "comparisons": comparisons
+        "comparisons": comparisons,
+        "exclusive": exclusive
     })
 
 
 def get_comparisons(cur, accessions: tuple):
-    
+
```

Comment:
Create `in_params` so we can re-use the bind variables for multiple SQL statements.

```suggestion
    in_params = ','.join('%s' for _ in accessions)
```